### ISSUE TITLE
markdown not being rendered

### ISSUE THREAD

**Issue by raedah** [vector-im/riotX-android#1248]
Double asterisks for bold as well as full urls for all included links are plainly visible. It appears markdown is not being rendered on the msgs. These msgs display fine on riot web.

**Comment by bmarty** [comment]
Is it still the case with RiotX v0.21.0 (which is not release yet)?

**Comment by DanieLoche** [comment]
> Is it still the case with RiotX v0.21.0 (which is not release yet)?

Yes. I am on RIotX v0.21.0 and I have no markdown working with messages sent from RiotX.
(I just have to Edit=>Send them from the desktop app afterwards to get the markdown code working)

**Comment by mpfaff** [comment]
@bmarty I've had the markdown rendering disabled twice between using the app. I haven't looked into reproducing the issue, but it gets really frustrating when I type out a long message with markdown formatting only for it to be escaped. I really think markdown should be enabled by default.
